lib/ide/parser/src/main.rs

```diff
@@ -1,5 +1,7 @@
 use parser::api::IsParser;
 
+use enso_prelude::default;
```

Comment:
If symbol is in prelude, we should import it whole (prelude::*) and place before other use declarations.
Also, we should prefer crate (parser's) prelude.